protection_profile.go

```diff
@@ -15,14 +15,18 @@ const (
 	ProtectionProfileAes128CmHmacSha1_32 ProtectionProfile = 0x0002
 	ProtectionProfileAeadAes128Gcm       ProtectionProfile = 0x0007
 	ProtectionProfileAeadAes256Gcm       ProtectionProfile = 0x0008
+
+	// These are not IANA registered, so used high values to avoid potential conflict
+	ProtectionProfileAes256CmHmacSha1_80 ProtectionProfile = 0xfff0
+	ProtectionProfileAes256CmHmacSha1_32 ProtectionProfile = 0xfff1
```

Comment:
In the RFC draft,
```
      SRTPProtectionProfile SRTP_AES256_CM_SHA1_80 = {0x00, 0x03};
      SRTPProtectionProfile SRTP_AES256_CM_SHA1_32 = {0x00, 0x04};
```
https://datatracker.ietf.org/doc/html/draft-ietf-avt-dtls-srtp-03#section-4.1.2

libsrtp in chromium code seems also using them: https://chromium.googlesource.com/chromium/deps/libsrtp/+/84122798bb16927b1e676bd4f938a6e48e5bf2fe/srtp/include/srtp.h#694

Reply:
Thanks for info! Updated this in code.